### PR TITLE
Documentation about setup in in air-gapped environment

### PR DESCRIPTION
**Pull request by dervoeti**
Fixes https://github.com/stackabletech/documentation/issues/270

### CODE REVIEW

File: modules/tutorials/pages/running_stackable_in_an_airgapped_environment.adoc [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+In the first two scenarios, you need to make sure that the nodes load the images from your local registry mirror. Again, there are a several ways to do this:
+
+* Specify the image repository in the CRDs (see https://docs.stackable.tech/home/nightly/concepts/product-image-selection#_custom_docker_registry["Custom docker registry"]) and in the values of the Helm charts when installing the operators (`helm install --set image.repository="my.custom.registry/stackable/nifi-operator" ...`).
+* If you use `containerd` as your container runtime, you can patch the `containerd` config on every node to use the mirrored registry instead of `oci.stackable.tech`. +
+Example: Create the file `/etc/containerd/certs.d/oci.stackable.tech/hosts.toml` with the following contents:
++
+[source,toml]
+----
+server = "http://10.7.228.12"
```
Comment (lfrancke) [inline modules/tutorials/pages/running_stackable_in_an_airgapped_environment.adoc:17]: I think this example could be clearer, not much, just a tiny introduction.

In this example you set up your own registry with the IP 10.7.228.12 etc.
And reading this I'd not be sure if I somehow need to tell it to override oci.stackable.tech or if this overrides all registry access requests

Comment (dervoeti) [reply modules/tutorials/pages/running_stackable_in_an_airgapped_environment.adoc]: Alright, clarified it a bit and also optimized the `hosts.toml` (removed unneded `server` line and specified `capabilities`). Tested it in my test environment, works.

**Review by lfrancke — APPROVED** [review]
I like it!
@fhennig do you also want to take another look?

**Comment by fhennig** [comment]
Thanks! I like it too, I think we can merge it as is.

I have an optional suggestion: Add a "Further reading" or "Read more" bit, where you link to offline image veriifcation. I think someone that reads this tutorial would be interested in that topic too, it helps with discoverability in the docs.

but LGTM!!

**Comment by dervoeti** [comment]
It was already there, but a bit hidden at the end. It's still at the end, but I made it more prominent.

**Pull request merged**